TestProject/input/User.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace input;
+
+use input\Base;
+use cellularAutomat\Board;
+use options\Getopt;
+
+/**
+ * The User can set a cell alive over a specific command in the command line.
+ *
+ * To do so an empty board is given where the user have to set the coordinates over the command line
+ * so the given cell coordinate is set alive but all the other cells are dead.
+ *
+ * @class User
+ */
+class User extends Base
+{
+    /**
+     * Prepares an empty board where the coordinates for the x/-axis has to be given, so the wished
+     * cell is set on alive but the rest cells are dead.
+     *
+     * CL-example: php gameoflife.php -i User 3,6
```

Comment:
I don't understand this. Why should the coordinates passed as parameters to the commandline-call? Isn't it that I have to type the coordinates when the fill-Board is running? You are using readline() so this says that you read the input from the keyboard when the input is being run. But why do you add an example of adding the coordinates as parameters?